server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectWriteAheadLogs.java

```diff
@@ -80,10 +80,10 @@
 
 public class GarbageCollectWriteAheadLogs {
   private static final Logger log = LoggerFactory.getLogger(GarbageCollectWriteAheadLogs.class);
+  private static final Map<HostAndPort,Long> firstSeenDead = new HashMap<HostAndPort,Long>();
 
   private final AccumuloServerContext context;
   private final VolumeManager fs;
-  private final Map<HostAndPort,Long> firstSeenDead = new HashMap<HostAndPort,Long>();
```

Comment:
@phrocker if a tserver came back online, the firstSeenDead map should be cleared [here](https://github.com/adamjshook/accumulo/blob/b3aa27c0add1b9ddfbed31b3d4ed24e4119aef69/server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectWriteAheadLogs.java#L320)